x-pack/plugins/apm/public/components/app/service_overview/service_overview_instances_chart_and_table.tsx

```diff
@@ -226,7 +226,7 @@ export function ServiceOverviewInstancesChartAndTable({
         <InstancesLatencyDistributionChart
           height={chartHeight}
           items={currentPeriodItems}
-          status={mainStatsStatus}
+          status={transactionType ? mainStatsStatus : FETCH_STATUS.SUCCESS}
```

Comment:
I'm not sure this is the way to go. Perhaps it's better to hide those visualisations entirely, and then communicate to the user that they don't have any transactions.

It sounds like you will solve [this issue](https://github.com/elastic/kibana/issues/118702#issuecomment-1376649981) simultaneously.

Reply:
Not sure about solving [118702](https://github.com/elastic/kibana/issues/118702) because the service that only have metrics will keep appearing in service maps and not in service overview

Reply:
I think it'll at least remedy this:

> If there are documents in metrics-* with a service.name the APM service inventory will show those services but nothing will show up on the overview when you click through.

If users click through to the service overview instead of seeing an empty screen they'll see a message explaining why they are not seeing anything.

Reply:
Either way, don't focus on https://github.com/elastic/kibana/issues/118702. I think your approach looks good.